django/thunderstore/social/api/experimental/views/current_user.py

```diff
@@ -14,6 +16,8 @@ class CurrentUserExperimentalApiView(APIView):
     Gets information about the current user, such as rated packages and permissions
     """
 
+    authentication_classes = [SessionAuthentication, BasicAuthentication]
```

Comment:
Feels odd to me if we have to add more auth classes here? What's the default authentication scheme in `APIView`, i.e. what's been used here so far?

Reply:
Oh I actually didn't notice that there was a separate auth class in the settings. I'll have to rework this.